server/views/pages/person/home.njk

```diff
@@ -1,47 +1,90 @@
 {% extends "../../partials/prisonerBannerLayout.njk" %}
 
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
 {% from "govuk/components/panel/macro.njk" import govukPanel %}
 {% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "hmpps/components/court-cases-release-dates/sub-navigation/macro.njk" import subNavigation %}
 
 {% set pageTitle = applicationName + " - person home" %}
 {% set pageId = "person-home" %}
 
+{% block beforeContent %}
+
+  {{ super() }}
+
+  {% block subNavigation %}
+    {% set navigation = {
+      activeSubNav: 'recalls',
+      adjustments: {
+        enabled: not profileBannerParams
+          },
+      courtCases: {
+        enabled: false
```

Comment:
have you got a ticket for this being role based enabled? I assume a user will be able to go to court cases if they have the correct role?